templates/bot.yaml

```
@@ -93,76 +93,89 @@ objects:
 - apiVersion: apps/v1
   kind: Deployment
   metadata:
-    name: postgresql
     labels:
-      app: "bot"
+      app: bot
+    name: postgresql
+    namespace: bot
   spec:
-    strategy:
-      type: Recreate
     replicas: 1
     selector:
       matchLabels:
         name: postgresql
+    strategy:
+      type: Recreate
     template:
       metadata:
-        name: postgresql
         labels:
           name: postgresql
+        name: postgresql
       spec:
-        volumes:
-        - name: pgdb-volume
-          persistentVolumeClaim:
-            claimName: bot-postgres-pvc
-        - name: pg-configs
-          configMap:
-            name: postgresql-configs
-        initContainers:
-        - name: pgsql-data-permission-fix
-          image: busybox
-          command: ["/bin/chmod","-R","777", "/var/lib/pgsql/data"]
-          volumeMounts:
-          - name: pgdb-volume
-            mountPath: "/var/lib/pgsql/data"
         containers:
-        - name: postgresql
-          image: "docker.io/manageiq/postgresql:10"
-          ports:
-          - containerPort: 5432
-          readinessProbe:
-            initialDelaySeconds: 60
-            tcpSocket:
-              port: 5432
-          volumeMounts:
-          - name: pgdb-volume
-            mountPath: "/var/lib/pgsql/data"
-          - name: pg-configs
-            mountPath: "/opt/app-root/src/postgresql-cfg/"
-          env:
+        - env:
           - name: POSTGRESQL_USER
             valueFrom:
               secretKeyRef:
-                name: postgresql-secrets
                 key: username
+                name: postgresql-secrets
           - name: POSTGRESQL_PASSWORD
             valueFrom:
               secretKeyRef:
-                name: postgresql-secrets
                 key: password
+                name: postgresql-secrets
           - name: POSTGRESQL_DATABASE
             valueFrom:
               secretKeyRef:
-                name: postgresql-secrets
                 key: dbname
+                name: postgresql-secrets
           - name: POSTGRESQL_MAX_CONNECTIONS
-            value: '1000'
+            value: "1000"
           - name: POSTGRESQL_SHARED_BUFFERS
             value: 1GB
+          image: docker.io/manageiq/postgresql:10
+          imagePullPolicy: IfNotPresent
+          name: postgresql
+          ports:
+          - containerPort: 5432
+            protocol: TCP
+          readinessProbe:
+            failureThreshold: 3
+            initialDelaySeconds: 60
+            periodSeconds: 10
+            successThreshold: 1
+            tcpSocket:
+              port: 5432
+            timeoutSeconds: 1
```

Original code before the reorder was:

```suggestion
           readinessProbe:
              initialDelaySeconds: 60
              tcpSocket:
                port: 5432
```